src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -651,6 +652,22 @@ private string[] GetResourceKeys(CancellationToken ct)
 			return resourceKeys;
 		}
 
+		private string RewriteResourceKeyName(StringBuilder builder, string keyName)
+		{
+			var firstDotIndex = keyName.IndexOf('.');
+			if (firstDotIndex != -1)
+			{
+				builder.Clear();
+				builder.Append(keyName);
+
+				builder[firstDotIndex] = '/';
```

Comment:
Previously we were replacing all `.`s with `/`s
Now we replace only the first `.`, not sure if that's intentional.

Reply:
It's intentional, yes. That's the way resources are looked up using ResourceLoader.